include/aspect/lateral_averaging.h

```diff
@@ -248,7 +248,7 @@ namespace aspect
       get_velocity_magnitude_averages(std::vector<double> &values) const;
 
       /**
-       * Compute a lateral average of the current sinking velocity.
+       * Compute a lateral average of the current rising velocity.
```

Comment:
You'll have to undo this change, and make a change sinking->rising in line 261 so that it matches the function.